modules/ztp-configuring-the-hub-cluster-for-backup-and-restore.adoc

```diff
@@ -0,0 +1,177 @@
+    kind: Policy
+---
+apiVersion: cluster.open-cluster-management.io/v1beta2
+kind: ManagedClusterSetBinding
+metadata:
+  name: default
+  namespace: default
+spec:
+  clusterSet: default
+----
+<1> If you apply the `cluster.open-cluster-management.io/backup: cluster-activation` label to `BareMetalHost` resources, the {rh-rhacm} cluster backs up those resources.
+You can restore the `BareMetalHost` resources if the active cluster becomes unavailable, when restoring the hub activation resources.
+
+. Apply the policy by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f BareMetalHostBackupPolicy.yaml
+----
+
+.Verification
+
+. Find all `BareMetalHost` resources with the label `infraenvs.agent-install.openshift.io`  by running the following command: 
++
+[source,terminal]
+----
+$ oc get BareMetalHost -A -l infraenvs.agent-install.openshift.io
+----
++
+.Example output
+[source,yaml]
+----
+NAMESPACE      NAME             STATE   CONSUMER   ONLINE   ERROR   AGE
+baremetal-ns   baremetal-name                      false            50s
+----
+
+. Verify that the policy has applied the label `cluster.open-cluster-management.io/backup=cluster-activation` to all these resources, by runing the following command:
++
+[source,terminal]
+----
+$ oc get BareMetalHost -A -l infraenvs.agent-install.openshift.io,cluster.open-cluster-management.io/backup=cluster-activation
+----
++
+.Example output
+[source,yaml]
+----
+NAMESPACE      NAME             STATE   CONSUMER   ONLINE   ERROR   AGE
+baremetal-ns   baremetal-name                      false            50s
+----
++
+The output must show the same list as in the previous step, which listed all `BareMetalHost` resources with the label `infraenvs.agent-install.openshift.io`.
+This confirms that that all the `BareMetalHost` resources with the `infraenvs.agent-install.openshift.io` label also have the `cluster.open-cluster-management.io/backup: cluster-activation` label.
++
+The following example shows a `BareMetalHost` resource with the `infraenvs.agent-install.openshift.io` label. 
+The resource must also have the `cluster.open-cluster-management.io/backup: cluster-activation` label, which was added by the policy created in step 1.
++
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  labels:
+    cluster.open-cluster-management.io/backup: cluster-activation
+    infraenvs.agent-install.openshift.io: value
+  name: baremetal-name
+  namespace: baremetal-ns
+----
+
+You can now use {rh-rhacm-title} to restore a managed cluster. 
+
+[IMPORTANT]
+====
+When you restore `BareMetalHosts` resources as part of restoring the cluster activation data, you must restore the `BareMetalHosts` status.
+The following {rh-rhacm} `Restore` resource example restores activation resources, including `BareMetalHosts`, and also restores the status for the `BareMetalHosts` resources:
+[source,yaml]
+----
+  apiVersion: cluster.open-cluster-management.io/v1beta1
```

Comment:
I think this should be left justified? (there are extra spaces in front of `apiVersion`)